maldoca/ole/oss_utils.cc

```diff
@@ -56,14 +56,19 @@ void InitSAXHandler() {
 }
 
 #if defined(_WIN32)
-void HandleConverterError(UConverter* conv, const char* encode_name,
+void CloseUConverter(UConverter* conv) {
+  ucnv_close(conv);
+  conv = nullptr;
```

Comment:
this doesn't assign the original ptr to null just the copy on the stack.  you will need to pass in either a ** or *&

Reply:
done. do u mind having another look?